src/LoginDialog/OidcUtilities.ts

```diff
@@ -24,10 +24,19 @@ export const generateOidcLoginRedirectUrl = (baseRedirectUrl: string, cId: strin
         'response_type=code',
         'scope=openid+email+profile'
     ].join('&');
-    
-    return `${baseRedirectUrl}?${oidcParameters}`
+
+
+    let delimeter = "?";
+    if (baseRedirectUrl.indexOf("?") >= 0) {
```

Comment:
you can use `endsWith()` if you just want to check for a trailing question mark. `indexOf` will check for the entire string.

Reply:
we want to check if there are existing query params, if so we want to add a & instead of starting with a ?

Reply:
maybe a .contains() method makes more sense? or should we add a comment to the code? i think it should be the same complexity on either search for the ? 😄  but maybe contains is clearer at first glance?

Reply:
@shaunmaharaj the ? may not be at the end of the string, in our case its something like `/realms/realm-id?profile=xyz` so we would want to have  `/realms/realm-id?profile=xyz&code=abc` instead of  `/realms/realm-id?profile=xyz?code=abc`

Reply:
Ah got it! I just added a suggestion below using indexOf then. You could trim this down to 1 line if you wanted using template literals. 

What's the reason for us not using URL though in the fiddle I posted?

Reply:
https://jsfiddle.net/u1p8fnbL/

Reply:
Not a good reason for not using URL, I had to do the same in another area, and I ust reused the logic. I can conver this to use the URL though.